endpoints/api/superuser_models_pre_oci.py

```diff
@@ -75,7 +75,7 @@ def get_repository_build(self, uuid):
         except model.InvalidRepositoryBuildException as e:
             raise InvalidRepositoryBuildException(str(e))
 
-        repo_namespace = build.repository_namespace_user_username
+        repo_namespace = build.repository.namespace_user.username
         repo_name = build.repository_name
```

Comment:
For the sake of consistency, would it make sense to also change the next line?
```
repo_name = build.repository.name
```

Reply:
A repo name and a repo namespace are 2 different things. The latter refers to the user/org name

Reply:
Updated to to user the correct attribute name.